services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/UserTenantServiceApplication.java

```diff
@@ -2,12 +2,21 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.PropertySource;
 
 @SpringBootApplication
+@PropertySource({"classpath:message.properties"})
```

Comment:
What is this for ?

Reply:
This is used to get the messages from properties file, that creates loose coupling. 
we got the message values from external file instead writing into code. 
we may change the value if required without interfering jar .
but if your requirement is different we may put the messages in code.